x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/test_connector_form.tsx

```diff
@@ -90,6 +95,7 @@ export const TestConnectorForm = ({
               }
               messageVariables={[]}
               actionConnector={connector}
+              executionMode={ActionConnectorMode.Test}
```

Comment:
Allowing connectors to determine whether they're being rendered by the test component